singularity_container/build_container.md

```diff
@@ -15,51 +15,22 @@ All other steps are as pointed out in the documentation linked above.
 
 Building the container using the provided `conda_rdkit2019.def` definition file requires only a few steps:
```

Comment:
container name should also be changed here

Reply:
Yep, thanks. This is one of the places I need to change. And the main readme should also get an appropriate entry now.